editor.planx.uk/src/@planx/components/Question/Public/Question.tsx

```diff
@@ -89,7 +90,7 @@ const Question: React.FC<IQuestion> = (props) => {
         definitionImg={props.definitionImg}
         img={props.img}
       />
-      <FormControl sx={{ width: "100% " }}>
+      <FormControl sx={{ ...contentFlowSpacing(theme), width: "100%" }}>
```

Comment:
In this example, we can't use the shorthand version as we're using the [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) (`...`) to also add in the `width` property.

Reply:
Appreciate this context 👍